index.mako

```diff
@@ -1124,6 +1124,40 @@ the commit will be aborted.
 [commit-msg]: https://git-scm.com/docs/githooks#_commit_msg
 [prepare-commit-msg]: https://git-scm.com/docs/githooks#_prepare_commit_msg
 
+
+_new in __VERSION__: pre-commit can be used to manage [post-checkout] hooks.
+
+To use `post-checkout` hooks with pre-commit run:
+
+```console
+$ pre-commit install --hook-type post-checkout`
+pre-commit installed at .git/hooks/post-checkout
+```
+
+`post-checkout` hooks can be used to perform repository validity checks,
+auto-display differences from the previous HEAD if different,
+or set working dir metadata properties. Since `post-checkout` doesn't operate
+on files, any hooks must set `always_run`:
+
+```yaml
+-   repo: local
+    hooks:
+    -   id: post-checkout-local
+        name: Post checkout
+        entry: bash -c ""
+        language: system
```

Comment:
I would put `# ...` in place of the `language / entry` here since you're really only trying to highlight the `always_run` and `stages` options